plinth/modules/tor/templates/tor.html

```diff
@@ -53,7 +53,11 @@
       {% endif %}
     </p>
 
-    {% include "diagnostics_button.html" with module="tor" %}
+    {% if status.is_running %}
+      {% include "diagnostics_button.html" with module="tor" %}
```

Comment:
Hi @SunilMohanAdapa , thank you for looking at this.
Maybe I don't understand this right, but I think you suggestion would still fail in the case of the include from the `users` module, see https://github.com/freedombox/Plinth/blob/master/plinth/modules/users/templates/users_list.html#L67. That is, unless we define there also a variable `app_is_running`, just for this button.

It seems to me we have to choose between two extremes: Either make the button choices "dumb", and select which one to show in the module/template that includes it (current solution). Or both button definitions are within the same file, and there's a unified variable in each including module that decides which one to show (sort of your suggestion, except we also need to add something in the Users module).
I'm still leaning towards my current solution; having two templates, and deciding in the including module, because that's where the variables are present, which makes it more readable imo.
But it's easy to convince me otherwise, maybe I'm seeing it wrong.